script/release.sh

```diff
@@ -33,7 +33,9 @@ function build_project() {
 	local libseccomp_ver='2.5.1'
 	local tarball="libseccomp-${libseccomp_ver}.tar.gz"
 	local prefix
+	local ldflags
 	prefix="$(mktemp -d)"
+	ldflags="-w -s"
```

Comment:
nit:
```suggestion
local ldflags="-w -s"
local prefix
prefix=$(mktemp -d)"
```

The `prefix` assignment is separated from declaration as otherwise it would hide an error from mktemp. There can be no error in assigning a string, so no need to do the same for `ldflags` assignment.

Reply:
@kailun-qin please address this one ^^^^

Reply:
Ah, I see, you already did that, but in a separate commit.